cognite/client/data_classes/workflows.py

```diff
@@ -96,7 +97,7 @@ def as_external_ids(self) -> list[str]:
 
 
 class WorkflowTaskParameters(CogniteObject, ABC):
-    task_type: ClassVar[Literal["function", "transformation", "cdf", "dynamic"]]
+    task_type: ClassVar[Literal["function", "transformation", "cdf", "dynamic", "subworkflow"]]
```

Comment:
This Literal is specified in three places, let's make it into a type alias then use it in the error message to avoid having to update it a fourth place 😄